<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калькулятор дробей (C++/Qt)</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-calculator">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_Display">
      <item>
       <widget class="QLabel" name="memoryIndicatorLabel">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="displayLineEdit">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0/1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_BS">
        <property name="toolTip">
         <string>Backspace</string>
        </property>
        <property name="text">
         <string>←</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_CE">
        <property name="toolTip">
         <string>Clear Entry</string>
        </property>
        <property name="text">
         <string>CE</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButton_C">
        <property name="toolTip">
         <string>Clear All</string>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_MC">
        <property name="toolTip">
         <string>Memory Clear</string>
        </property>
        <property name="text">
         <string>MC</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_9">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="pushButton_divide">
        <property name="toolTip">
         <string>Деление</string>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="pushButton_sqrt">
        <property name="toolTip">
         <string>Возведение в квадрат</string>
        </property>
        <property name="text">
         <string>x²</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_MR">
        <property name="toolTip">
         <string>Memory Recall</string>
        </property>
        <property name="text">
         <string>MR</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="pushButton_multiply">
        <property name="toolTip">
         <string>Умножение</string>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="pushButton_reciprocal">
        <property name="toolTip">
         <string>Обратная дробь</string>
        </property>
        <property name="text">
         <string>1/x</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_MS">
        <property name="toolTip">
         <string>Memory Store</string>
        </property>
        <property name="text">
         <string>MS</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="pushButton_subtract">
        <property name="toolTip">
         <string>Вычитание</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <spacer name="horizontalSpacer_3_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_Mplus">
        <property name="toolTip">
         <string>Memory Add</string>
        </property>
        <property name="text">
         <string>M+</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton_0">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButton_sign">
        <property name="toolTip">
         <string>Сменить знак</string>
        </property>
        <property name="text">
         <string>+/-</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="pushButton_fraction">
        <property name="toolTip">
         <string>Ввод знаменателя</string>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="pushButton_add">
        <property name="toolTip">
         <string>Сложение</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QPushButton" name="pushButton_equals">
        <property name="toolTip">
         <string>Вычислить</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" colspan="2">
       <spacer name="horizontalSpacer_top_right">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuHistory">
    <property name="title">
     <string>История</string>
    </property>
    <addaction name="actionLookHistory"/>
    <addaction name="actionClearHistory"/>
   </widget>
   <addaction name="menuHelp"/>
   <addaction name="menuHistory"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionLookHistory">
   <property name="text">
    <string>Показать</string>
   </property>
  </action>
  <action name="actionClearHistory">
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
